fcl/dunefd/reco/reco_supernova_dune10kt_1x2x6.fcl

```diff
@@ -1,3 +1,34 @@
-#include "standard_reco_dune10kt_1x2x6.fcl"
+#include "standard_reco_dune10kt.fcl"
 
+services: 
+{
+    @table::services
+    @table::dunefd_1x2x6_reco_services
+}
+
+physics.producers:
+{
+    @table::physics.producers
+    wclsmcnfsp: @local::dune10kt_1x2x6_mc_nfsp
+}
+
+services.DetectorPropertiesService.NumberTimeSamples: 6000
+services.DetectorPropertiesService.ReadOutWindowSize: 6000
+
+physics.producers.gaushit.CalDataModuleLabel: "wclsmcnfsp:gauss"
+physics.producers.gaushit.HitFilterAlg.MinPulseHeight: [ 2, 2, 2 ]
 physics.producers.spsolve.MinNHits: 2 #lower the threshold for low energy interactions
+physics.producers.pmtracktc.WireModuleLabel: "wclsmcnfsp:gauss"
+
+physics.reco: [
```

Comment:
Same here.

Reply:
@lpaulucc, I did the same here as for the 1x2x2 case above.